src/components/ProjectSettingsTabs/GithubCommitQueueTab/getFormSchema.ts

```diff
@@ -178,6 +183,65 @@ export const getFormSchema = (
           },
         },
       },
+      commitQueue: {
+        type: "object" as "object",
+        title: "Commit Queue",
+        properties: {
+          enabled: {
+            type: ["boolean", "null"],
+            oneOf: radioBoxOptions(
+              ["Enabled", "Disabled"],
+              repoData?.commitQueue?.enabled
+            ),
+          },
+          message: {
+            type: "string" as "string",
+            title: "Commit Queue Message",
+          },
+          mergeMethod: {
+            type: ["string"],
+            title: "Merge Method",
+            oneOf: [
+              {
+                type: "string" as "string",
+                title: "Squash",
+                enum: ["squash"],
+              },
+              {
+                type: "string" as "string",
+                title: "Merge",
+                enum: ["merge"],
+              },
+              {
+                type: "string" as "string",
+                title: "Rebase",
+                enum: ["rebase"],
+              },
+              ...insertIf(repoData, {
+                type: "string" as "string",
+                title: `Default to Repo (${repoData?.commitQueue?.mergeMethod})`,
+                enum: [""],
+              }),
+            ],
+          },
+          patchDefinitions: {
+            type: "object" as "object",
+            title: "Commit Queue Patch Definitions",
```

Comment:
I am able to press save with an empty tag or etc; although I do get this upset backend error but I wonder if it's possible have the UI prevent trying to do this in the first place.
![image](https://user-images.githubusercontent.com/26798134/152256912-1e975343-0070-45aa-97fc-2195cc43fb8f.png)

Reply:
Thanks, added the same validation that's used for other aliases on this page 👍